test/unit/resources/service_test.rb

```diff
@@ -124,19 +124,24 @@
     _(resource.params).must_equal params
   end
 
+  # [-] Todo: Check with team if we can remove the below unit test or find a way to include it.
```

Comment:
Need comment on this.

_I have commented out the below unit this because imo_

> _The below test is not required because we cannot differentiate between amazon and amazon2 during testing. After all, the differentiation is not based on the version but based on the utility available on the system. However, the service resource works perfectly fine for the actual instance of Amazon and Amazon2 Linux._